lib/revisions/__tests__/integration.js

```diff
@@ -47,6 +51,16 @@ test('authentication / success', async t => {
   t.deepEqual(res.body, {name: 'ACME'})
 })
 
+test('admin authentication / success', async t => {
+  const server = await getApp({clients: [fakeClient]})
```

Comment:
Il n'est pas nécessaire d'attribuer des clients.

```suggestion
  const server = await getApp()
```

Reply:
Le test est KO sur GitHub s'il n'y a pas de client, parce qu'il s'attend à avoir un fichier `client.yml` s'il n'y a pas d'argument.
Du coup en local sans l'argument le test est ok, sur le repo distant, lors du lancement des jobs, le test échoue.